doc/ssl/SSL_CTX_set_tlsext_status_cb.pod

```diff
@@ -16,8 +21,10 @@ Certificate Status Request functions
  long SSL_CTX_set_tlsext_status_arg(SSL_CTX *ctx, void *arg);
 
  long SSL_CTX_set_tlsext_status_type(SSL_CTX *ctx, int type);
+ long SSL_CTX_get_tlsext_status_type(SSL_CTX *ctx);
 
  long SSL_set_tlsext_status_type(SSL *s, int type);
+ long SSL_get_tlsext_status_type(SSL *s);
```

Comment:
I'd be happy if we decided to not use SSL_ctrl and turn them into real functions for all four of these. Though if we do that then we should probably point out the return type change for SSL_set_tlsext_status_type() somewhere.